### ISSUE TITLE
some apps fail to install

### ISSUE THREAD

**Issue by GoogleCodeExporter** [HopelessFantasy/openetna#186]
```
Did you search through the other issues to make sure that this issue has
not been already reported?
yes
What steps will reproduce the problem?
1.download and install google sky Map
2.download and install android-vnc-viewer
3.

What is the expected output? What do you see instead?
Downloads and attempts to install but get a error back
Installation unsuccessful

What version of the boot.img and system.img are you using?
Boot.img V2.1.1 Boot.img V2.1.1 systemV6.0-rc5.7z

Can you provide logcat output during that error?
tell me where to find

Please provide any additional information below.


```

Original issue reported on code.google.com by `bsmith.k...@gmail.com` on 6 Oct 2010 at 11:26
- Blocked on: #196
- Merged into: #196
